gradle.properties

```diff
@@ -2,11 +2,11 @@
 # -> https://www.jetbrains.org/intellij/sdk/docs/reference_guide/intellij_artifacts.html
 pluginGroup=com.github.intheclouddan.intellijpluginld
 pluginName=intellij-plugin-ld
-pluginVersion=1.0.4-idea23
+pluginVersion=1.0.4-idea24
 pluginSinceBuild=222
-pluginUntilBuild=232.*
+pluginUntilBuild=2024.*
```

Comment:
This change gives me pause -- did the build number format change as part of this release?

Reply:
Thanks for catching that

Reply:
Well that fixed the error I was having 🤦🏽